src/hotspot/share/cds/filemap.cpp

```diff
@@ -462,6 +462,15 @@ bool SharedClassPathEntry::validate(bool is_class_path) const {
     } else {
       FileMapInfo::fail_continue("A jar file is not the one used while building"
                                  " the shared archive file: %s", name);
+      if (!log_is_enabled(Info, cds)) {
+        log_warning(cds)("A jar file is not the one used while building"
+            " the shared archive file: %s", name);
```

Comment:
The string literal should be factored out and reused in both places.

Reply:
Fixed.